spec.bs

```diff
@@ -3532,6 +3603,11 @@ The <dfn for=Navigator method>updateAdInterestGroups()</dfn> method steps are:
           set |ig|'s [=interest group/trusted bidding signals keys=] to |value|.
         1. Otherwise, jump to the step labeled <i><a href=#abort-update>Abort update</a></i>.
 
+        <dt>"`trustedBiddingSignalsSlotSizeMode`"
+        1. If |value| is one of "none", "slot-size", or "all-slots-requested-sizes", set |ig|'s
+           [=interest group/trusted bidding signals slot size mode=] to |value|.
+        1. Otherwise, set [=interest group/trusted bidding signals slot size mode=] to "none".

```

Comment:
just to double check, so when the update IG provides an invalid value, we update the IG's sizeMode to "none", instead of keeping its old value, correct?

Reply:
Correct.